### DISCUSSION TITLE
openmoonray-1.6.0.0

### DISCUSSION THREAD

**Discussion by jlanz** [dreamworksanimation/openmoonray#158] [Announcements]
### Apple silicon initial support
* M-series macs with macOS Sonoma (macOS 14)
* MoonRay XPU mode using Metal with UMA
* Intel Open Image Denoise (OIDN) using Metal
* hdMoonray Hydra render plugin
* All command-line tools, including moonray and moonray_gui
* Arras for multi-machine rendering
### Build Improvements
* Installing cgroups now optional
* Fewer compiler warnings
* Cleaned up redundant CMake files
* Misc build cleanup/improvements
### Bug Fixes
* Fixed lib_rendering_pbr unit test resource issue
* Fixed pixel sample map could generate 0 samples
* Fixed mcrt_computation crash related to visibility aov
* Fixed reloading textures did not trigger a render update (hdMoonray)
* Fixed ImageMap missing file error if use_default_color_when_missing is true
* Fixed resume render regression
* Fixed improperly applied prim attr instance overrides
* Fixed crash/overflow in memory pool code
* Fixed uninitialized motion blur vars for primitives
* Fixed potential render prep hang and crash
* Fixed XPU handler depth bug
* Fixed several bugs with fallback to vector mode from XPU mode
* Fixed USDGeometry crashing in XPU mode when geometry is empty
* Fixed memory pool crash when XPU queue sizes are increased
* Fixed random sampling issue that was causing fireflies in vector mode
* Fixed implicit 64->32 bit truncations
* Fixed -size parameter in moonray_gui
* Fixed crash when both sets of velocity data and acceleration data are present
* Fixed mcrt_computation shutdown issue related to pthread-condition-wait
* Fixed moonray command -layer option
### Features
* Improved adaptive light sampling
* `rdla_filter` tool improvements
* New [usd_mipmap_images](https://docs.openmoonray.org/user-reference/tools/usd_mipmap_images/) tool for handling USD scenes with non-mipped/non-tiled textures
* Added OIDN engine to hdMoonray denoiser
* New FisheyeCamera
* Added inside-out projection for SphericalCamera
* Misc performance improvements
* Added cryptomatte_output_p0 to RenderOutput
* Enabled multithreaded GPU access
* Improved CPU affinity control
* Automatically calculate max v dimension for UDIM textures
* Added "observer direction" map type to AttributeMap
* Improved memory stats for vector and XPU modes
* Added tessellation memory usage stats
* Added light sampling stats
### Other
* Added status messages for Houdini Solaris (hdMoonray)
* Change default texture coordinate name to 'st'
* Replaced deprecated tbb::atomic with std::atomic
* Create sub directories for RDL2 file path if they don't exist (hdMoonray)
* Improved SceneClass attribute documentation
* Arras and Rndr (debug mode) support now in two separate plugins (hdMoonray)
* Plugin settings cleanup (hdMoonray)
* Removed multi-layer deep image output support
* Vector math lib macro/function cleanup
* Remove unnecessary MemPool templating
* Removed Centos 7 build instructions
### Community Contributors
* Alexander Fuller
* Yoshiaki Senda

<hr /><em>This discussion was created from the release <a href='https://github.com/dreamworksanimation/openmoonray/releases/tag/openmoonray-1.6.0.0'>openmoonray-1.6.0.0</a>.</em>

**Comment by mpjohn** [comment]
Does this version work out of the box with any particular version of Houdini?
